CHANGES.md

```diff
@@ -1,38 +1,97 @@
+# 2.17.0 - 26 Sep 2024
+
+## Features
+
+* optionally install Scenario with `ops[testing]` and expose the names in ops.testing in (#1381)
+* add a JujuVersion property for Pebble log forwarding to Loki in (#1370)
+* expand the secret ID out to the full URI when only given the ID in (#1358)
+* pre-emptively raise `InvalidStatusError` instead of waiting for Juju:
+    * make it an error to call `CollectStatusEvent.add_status` with error or unknown in (#1386)
+    * document and validate settable status values in `_ModelBackend.set_status` in (#1354)
+
+## Fixes
+
+* rework `ops.main` type hints to allow different flavours (callable class) in (#1345)
```

Comment:
I feel like the impact of this won't be understood, even though this is the PR title and it does do what it says.

Maybe:

```suggestion
* Change ops.main() so that you don't need to `type: ignore` it (#1345)
```